book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [OPTIONAL] It will be nice to also set a `setter method` for the `rentals` instance variable, as it is also modified in the `Book` class. You can replace the two lines above with: 
```
attr_accessor :title, :author, :rentals

```
Please kindly fix that.

Reply:
Hello @t-yanick, thanks for the tip 👍  I will fix it right away